packages/components/src/Dialog/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {
+  ComponentProps,
+  FC,
+  ReactElement,
+  cloneElement,
+  useCallback,
+  useMemo,
+  useState,
+} from 'react';
+
+import Modal from 'react-native-modal';
+
+import Box from '../Box';
+
+import DialogDefault from './components';
```

Comment:
参考建议不一定需要使用，这里可以直接引用文件里的类型
```suggestion
import type { ContentProps } from './components/Content';
import type { FooterButtonProps } from './components/FooterButton';
```

Reply:
其实我还想把 DialogDefault 暴漏出去。自定义 Dialog 时候可以用现成的部分。